purchase_stock_picking_return_invoicing/views/purchase_view.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="purchase_order_form" model="ir.ui.view">
+        <field name="name">purchase.order.form</field>
+        <field name="model">purchase.order</field>
+        <field name="inherit_id" ref="purchase.purchase_order_form"/>
+        <field name="arch" type="xml">
+            <button name="action_view_invoice" position="after">
+                <button type="object" name="action_view_invoice_refund"
+                        class="oe_stat_button"
+                        icon="fa-pencil-square-o">
+                    <field name="invoice_refund_count"
+                           widget="statinfo"
+                           string="Refunds"
+                           attrs="{'invisible': [('state', 'in', ('draft', 'sent', 'to approve'))]}"
+                    />
+                </button>
+            </button>
+            <xpath expr="//field[@name='order_line']/tree/field[@name='qty_received']" position="after">
+                <field name="qty_returned"
+                       invisible="not context.get('show_purchase', False)"/>
```

Comment:
This doesn't appear to work in Odoo 12. Instead, we can mirror the `attrs` used in core's Purchase Order form.

```suggestion
                       attrs="{'column_invisible': [('parent.state', 'not in', ('purchase', 'done'))]}"/>
```